app/controllers/application_controller.rb

```diff
@@ -40,4 +40,10 @@ def alert_and_log(message, error = nil)
     Rails.logger.error message + error&.backtrace.to_s
     flash[:danger] = message
   end
+
+  def show_geocode_message(version)
+    unless ENV.fetch('RAILS_ENV') == 'test'
```

Comment:
Code shouldn't be written just for the test environment. Is there a way to fix the tests instead of changing the code to make the tests pass?

Reply:
removed the references using ENV.fetch('RAILS_ENV') == 'test'